### PR TITLE
annotations: Toolbar icons adjustment

### LINKED ISSUE #4155: [Annotations] Toolbar icon measurement adjustment
The toolbar icon seems a little off alignment.  I think is because of the pen icon a little slimmer than the rest of icon. Also, the color button is a bit large, it creates an uneven space.   
I've adjusted the pen icon, and we may need to change the color circle to a smaller size 19x19 ( not included the stroke )  it should be able to slove the problem. Thanks!
**Pen icon**
https://github.com/mozilla/testpilot-assets/blob/master/Firefox_Screenshots/Annotation/Assets/annotation-pen.svg
**Meaturment spec**: 
https://mozilla.github.io/testpilot-assets/Firefox_Screenshots/Annotation/Annotation-icon-spacing/

Image: 
screen shot 2018-02-27 at 5 13 12 pm

### PR DESCRIPTION
**Pull request by niharikak101**
Fix #4155

### CODE REVIEW

**Comment by chenba** [comment]
@niharikak101 can you rebase this with master?  The failing tests are being skipped now.

**Comment by niharikak101** [comment]
@chenba done!

File: static/img/annotation-pen-active.svg [hunk 1]
```diff
@@ -1 +1,37 @@
...
+						<path id="path-1" class="st3" d="M2.7,2L1,8.5c-0.2,0.6-0.1,1.2,0,1.7c0.2-0.1,0.5-0.2,0.7-0.2l4.1,0c0.2,0,0.5,0.1,0.7,0.2
+							c0.1-0.5,0.2-1.1,0-1.7L4.8,2C4.6,1.1,4.2,0.7,3.8,0.7C3.3,0.7,2.9,1.1,2.7,2z"/>
+					</g>
+				</mask>
+				<polygon id="Fill-3" class="st4" points="-3.6,15.8 11.1,15.8 11.1,-4.8 -3.6,-4.8 				"/>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>
```
Comment (chenba) [inline static/img/annotation-pen-active.svg]: @niharikak101 If you `npm run svgo`, it should optimize both of these svgs.

**Comment by chenba** [comment]
[As Ian noticed before, the svgs changed _again_.](https://github.com/mozilla-services/screenshots/pull/3978#issuecomment-358756031)  What if we repeatedly apply `svgo` on them?  🤷‍♂️

**Pull request merged**